tests/bsim/bluetooth/audio/src/bap_scan_delegator_test.c

```diff
@@ -267,6 +268,14 @@ static int pa_sync_req_cb(struct bt_conn *conn,
 
 	if (past_avail) {
 		err = pa_sync_past(conn, state, pa_interval);
+		if (err == 0) {
+			err = bt_bap_scan_delegator_set_pa_state(state->recv_state->src_id,
+								 BT_BAP_PA_STATE_INFO_REQ);
```

Comment:
Very nice that you've added this support. Have you verified that it getting triggered correctly?